python/plugins/processing/algs/qgis/TilesXYZ.py

```diff
@@ -183,7 +201,9 @@ def generate(self, writer, parameters, context, feedback):
         settings.setLayers(self.layers)
         settings.setOutputDpi(dpi)
         if self.tile_format == 'PNG':
-            settings.setBackgroundColor(QColor(Qt.transparent))
+            color = QColor()
```

Comment:
This option looks odd - if you set it to 50%, it forces a 50% transparent black background? What's the use case here?

Reply:
I suggest a checkbox to have it either transparent or not.

Reply:
Actually I'd prefer to completely defer this, and have a dedicated "color" parameter type for it. If we add anything before this, it gets locked in to stable API forever...

Reply:
or we can take background color from the current project